x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/enable.ts

```diff
@@ -119,6 +119,7 @@ export default function createEnableAlertTests({ getService }: FtrProviderContex
                 .auth(user.username, user.password)
                 .expect(200);
               expect(typeof updatedAlert.scheduled_task_id).to.eql('string');
+              expect(updatedAlert.execution_status.status).to.eql('pending');
```

Comment:
Note to self: this is flaky in the scenario the rule finished running before the API call is made.

Reply:
Had to remove the integration test: https://github.com/elastic/kibana/pull/98135/commits/cfb1c259ccacef9ccc08f4ac3578e9c328b76ece. It will always be a race condition between enabling the rule and checking if the execution status is `pending`. So we'll have to rely on unit tests.